main.tf

```diff
@@ -3,6 +3,23 @@ locals {
     managed_by = "terraform"
     module     = "materialize"
   })
+
+  # Disk support configuration
+  disk_config = {
+    install_openebs           = var.enable_disk_support ? lookup(var.disk_support_config, "install_openebs", true) : false
+    run_disk_setup_script     = var.enable_disk_support ? lookup(var.disk_support_config, "run_disk_setup_script", true) : false
```

Comment:
Could we just base this one entirely on `var.enable_disk_support` and not have this extra var?

Reply:
Yes, the idea was that users only set `enable_disk_support`, and the rest use defaults. I added `disk_support_config` just for extra flexibility if we ever need to override things. Happy to simplify if we want to keep it more opinionated though, up to you!

Reply:
This part of the PR is more or less a copy and paste from the AWS implementation.

Reply:
ok.... let's just keep this